### ISSUE TITLE
[LC] Establish an MVP hypothesis for LC taxonomy

### ISSUE THREAD

**Issue by oksana-c** [department-of-veterans-affairs/va.gov-cms#2542]
## Goals
* review and understand the purpose / structure / usage of Topics and Audience taxonomies in the context of the Learning Center
* review opportunities for reusability in other VA products
* develop an MVP set of terms that will support extensible taxonomy vocabularies in the CMS and will satisfy veteran-facing as well as editor-facing features

## Hypotheses

1. _Unified VA-wide Topics and Audience taxonomy vocabularies that consist of unambiguous, intuitive, plain language terms are able to satisfy the majority of use cases for upcoming VA products._
1. _An authoritative, restricted list of terms supports easy retrieval of content by veterans as well and content editors._ 
1. _MVP set of terms for both Topics and Audience vocabularies will satisfy short-term needs of the Learning Center, while avoiding tech debt that will incur if the full taxonomy is implemented immediately [as described 08/10](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/template-requirements.md#topic-tags)._

## VA-wide taxonomy requirements

- reusability across multiple products
- support rich search
- support short-term Learning Center needs

## Acceptance Criteria
- [x] the CMS team has connected with VA stakeholders that are involved in development of VA products that may inform Topics and Audience taxonomy composition
- [x] MVP taxonomy terms for Topics vocabulary are identified
- [x] MVP taxonomy terms for Audience vocabulary are identified

**Comment by oksana-c** [comment]
We've had an opportunity to conduct initial research for the Learning Center MVP and gain some insight on the use of tags. 
A group of interested parties discussed research observations with a focus on opportunities to make Topics and Audience taxonomies re-usable for a larger set of VA products beyond the Learning Center.

Notes are available in this document https://docs.google.com/document/d/1Wu-dA_eGLOAOjz3G6iN9u2-siakWlZ5uzHfd3X86WS8

**TL;DR:**

* The scope of the Learning Center content boundaries is still largely unclear.
* Topics and Audience taxonomy vocabularies need to represent a unified authoritative source of VA terms and satisfy broad application use cases amongst VA products.
* The current state of Topics and Audience taxonomy is very much in play and refinements to structure and labels are expected within the next two sprints or so.
* The CMS team recommends implementing an MVP set of terms so as to unblock Learning Center template development and to prevent accumulation of technical debt.

---

#### Why implement an MVP set of terms vs. a full set as outlined in [template requirements](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/template-requirements.md#topic-tags)?

* many terms require label revisions. Efforts are underway to solidify terms in both vocabularies. 
* every term receives an URL alias upon creation in the CMS. This URL alias will be used in many places within the Learning Center to create connections to related content. When the term name changes, the alias also needs to change. The CMS team, Content Editors and FE engineers will have to spend time coordinating redirects/updates for links that exist in multiple places for each updated term
* FE team may need to modify template queries every time term label / URL changes
* If one term is deemed ineffective and is replaced by another (not renamed), the content tagged with the old term will need to be retagged with a new term. While there are multiple possibilities to provide content editors with UI to perform "retag" action, this functionality is not planned within the Learning Center MVP scope and will have to be performed in a manual fashion, which is prone to user error.
* Changing term labels often make for a poor content editor experience since users need to re-learn the meaning of a term. As a result, content will be at risk of mislabelling.

## Proposal for MVP Topics and Audience taxonomy terms

The CMS team recommends to start Topics and Audience taxonomy with a minimal set of terms that are unlikely to change in order to avoid technical debt issues described above.

### Topics (MVP set of tags)

1. Burials and memorials
1. Careers and employment
1. Disability
1. Education and training
1. Health care
1. Housing assistance
1. Life insurance
1. Pension
1. VA Account and Profile

Hold on implementing  the rest of Topic terms, since the [upcoming card-sort](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/discovery-and-research/card-sort-research-plan.md) may redefine the majority of labels, specifically "General" group.

### Audience

### Beneficiaries (MVP set of tags)

* Veterans
   * Elderly Veterans
   * Former prisoners of war
   * Gulf War Veterans
   * Incarcerated Veterans
   * Korean War Veterans
   * LGBTQ Veterans
   * Minority Veterans
   * Native American Veterans
   * Vietnam War Veterans
   * Women Veterans
   * World War II Veterans
* Family members and caregivers
* Service members

### Non-beneficiaries

Hold off on implementing "Non-beneficiaries" terms, since the [upcoming card-sort](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/discovery-and-research/card-sort-research-plan.md) may redefine the majority of labels.

**Comment by jenniferlee-dsva** [comment]
@oksana-c - We on the government side cannot access Google doc links. Can you please provide it as a Word doc or as a MD file in Github.

**Comment by oksana-c** [comment]
@jenniferlee-dsva , thank you for letting me know. I moved notes [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/discovery-and-research/topics-audience-taxonomy.md).

**Comment by jenniferlee-dsva** [comment]
@oksana-c - I made a couple of minor edits ^ in the beneficiary audience list. 

- We're testing 3 variations for that elderly Veteran category.
- Family members (deleted the "and caregivers")

**Comment by oksana-c** [comment]
new information was received from @jenniferlee-dsva on Aug 14. We'll schedule a follow up conversation wrt taxonomy in Sprint 13 (aug 17-28)

**Comment by oksana-c** [comment]
Taxonomy definition continues in [Learning Center taxonomy/gov brainstorm MURAL](https://app.mural.co/t/vagov6717/m/vagov6717/1597845145225/5e258a50346775afe0125cd5126bce952fde5d94). 

Next session scheduled for Tue, September 1 - 2:30pm ET with @jenniferlee-dsva @rtwell @kevwalsh @oksana-c .

**Comment by oksana-c** [comment]
MVP Topics and Audience taxonomy was developed in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/2795 and https://github.com/department-of-veterans-affairs/va.gov-cms/issues/2562 . 

Topics taxonomy vocabulary was populated with MVP set of terms described in #2795 

On September 9 , during a discussion about LC content URL alias patterns and breadcrumb IA, the group identified it may be best to split LC Category terms from the Topics vocabulary so as to keep Topics vocabulary focused on content tags instead of providing IA capabilities.

The group is scheduled to meet Sep 16 to revisit URL alias patterns, breadcrumb and taxonomy structure - https://github.com/department-of-veterans-affairs/va.gov-cms/issues/2818#issuecomment-689799120
